doc/autoscale/experiment/mnist/train_ft.py

```diff
@@ -0,0 +1,169 @@
+from PIL import Image
+import numpy as np
+import paddle.v2 as paddle
+import paddle.v2.dataset.common as common
+import os
+import sys
+import glob
+import pickle
+
+
+# NOTE: must change this to your own username on paddlecloud.
+USERNAME = "yanxu05@baidu.com"
```

Comment:
Can we put `mnist` dataset to the public folder so we don't need this hardcoded username anymore?

Reply:
Done.